client/src/App.js

```diff
@@ -29,13 +29,55 @@ function App() {
         <section className="app-info"></section>
         <section className="your-pick"></section>
         <section className="palette"></section>
-        <footer className="bottom"></footer>
         {/* <img src={logo} className="App-logo" alt="logo" /> */}
-
-         main
-        <p class="heading-font">{ !data ? "Loading..." : data}</p>{" "}
-
+        main
+        <p class="heading-font">{!data ? "Loading..." : data}</p>{" "}
         {/* If no data message, then it will load otherwise, there is a data message */}
+        <footer className="fixed bg-gray-800 w-full bottom-0 items-center flex flex-full flex-col py-20">
+          <div class="flex space-x-10 ...">
```

Comment:
What does `...` do here?

Reply:
The three dots do nothing. Do you want me to add address for each media too or only for line 37.

Reply:
Just line 37!